runtime/packages/models/assembly/block-hash.ts

```diff
@@ -0,0 +1,68 @@
+import { Hash, CompactInt, Bytes } from 'as-scale-codec';
+import { Utils } from '@as-substrate/core-utils';
+import { DecodedData } from '.';
+
+export class BlockHash{
+    /**
+     * the content/data of the BlockHash
+     */
+    public data: Map<CompactInt, Hash>;
```

Comment:
Instead of using Map for the CompactInt and Hash wouldn't it be better to have
`blockNumber` and `blockHash` as properties?